VERSION

```diff
@@ -1 +1 @@
-4.0.0
+4.0.1
```

Comment:
Hi, @tony-romanovych. Could you update it to 4.0.5, please? Thank you.

Reply:
It's 4.0.6 for now.

Reply:
Done.